device/common/include/traccc/finding/device/get_sort_key_value.hpp

```diff
@@ -0,0 +1,32 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2024 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s).
+#include "traccc/edm/device/sort_key.hpp"
+#include "traccc/edm/track_candidate.hpp"
+
+namespace traccc::device {
+
+/// Function used for fitting a track for a given track candidates
+///
+/// @param[in] globalIndex   The index of the current thread
+/// @param[in] params_view   The input parameters
+/// @param[out] keys_view    The key values
+/// @param[out] ids_view     The param ids
+///
+TRACCC_HOST_DEVICE inline void get_sort_key_value(
```

Comment:
I don't like this function name. :frowning:  This is absolutely subjective, true, but how about `fill_sort_keys`? Just something with `fill` would sound better to me than `get`. :thinking:

Reply:
OK I used `fill_sort_keys`